### PR TITLE
docs(src/main): add documentation to kernel.js

### PR DESCRIPTION
**Pull request by danoventa**
docs(src/main): add documentation to kernels

continues issue #838

### CODE REVIEW

**Comment by captainsafia** [comment]
Yay! Great to see a PR for this in.

Can you edit the commit messages and PR titles so that they conform to the convention we follow as specified in CONTRIBUTING.md?

**Comment by danoventa** [comment]
Oh, thought I did, can do!

File: src/main/kernels.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function ipyKernelTryObservable(env) {
 /**
   * condaInfoObservable executes the conda indo --json command and maps the
   * result to an observable that parses through the environmental informaiton.
```
Comment (jdetle) [inline src/main/kernels.js:25]: Can you fix this typo? Thanks

File: src/main/kernels.js [hunk 2]
```diff
@@ -83,9 +83,9 @@ export function createKernelSpecsFromEnvs(envs) {
 }
 
 /**
-  * condaKernelObservable generates an observable containing the supported langauage
+  * condaKernelObservable generates an observable containing the supported language
   * environmental elements.
-  * @return {Observable}  emits supported langauage elements
+  * @return {Observable}  emits supported language elements
```
Comment (jdetle) [inline src/main/kernels.js]: I'd love to see `{Observable} - Emits supported language elements.`

**Review by jdetle — COMMENTED** [review]
Awesome, thanks so much!

File: src/main/kernels.js (hunk 2 above)
Comment (captainsafia) [reply src/main/kernels.js]: It should actually be `Supported language elements` sine it describes a data type not an action.

Comment (jdetle) [reply src/main/kernels.js]: Thanks Safia!

**Comment by danoventa** [comment]
Alrighty, I went through some of the older documents to match this style as well. Thanks!

**Comment by jdetle** [comment]
Do you think this is ready for removal of the WIP tag @temogen?

**Comment by danoventa** [comment]
Hey John, I believe it's done, is there a guide to PR tags?

**Comment by jdetle** [comment]
Was just asking you if you could change the title! But this looks good to me too!

**Comment by jdetle** [comment]
Congrats on a successful PR! Hopefully many more to come!

**Comment by danoventa** [comment]
Thanks! Looking forward to getting more in!

**Pull request merged**